x-pack/plugins/ml/server/models/data_recognizer/modules/sample_data_ecommerce/manifest.json

```diff
@@ -1,7 +1,7 @@
 {
   "id": "sample_data_ecommerce",
   "title": "Kibana sample data eCommerce",
-  "description": "Find anomalies in eCommerce total sales data",
+  "description": "Find anomalies in eCommerce total sales data.",
```

Comment:
Adding the period makes this consistent with the other wizard cards on the page, but note that a lot of the descriptions in the other modules still lack an ending period. Is there a reason why you didn't add the period to the other manifest files too?

![image](https://user-images.githubusercontent.com/7405507/80962008-19145e00-8e04-11ea-850a-2e46d36999cf.png)

Reply:
Good point.  I'll add the change in other manifests.